tests/test_historical_price_transformer.py

```
@@ -0,0 +1,68 @@
+import unittest
+from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)